src/richie/apps/search/defaults.py

```diff
@@ -4,6 +4,20 @@
 from django.conf import settings
 from django.utils.translation import gettext as _
 
+# The React i18n library only works with ISO15897 locales (e.g. fr_FR)
+# Django also supports ISO639-1 language codes without a region (e.g. fr) which is sufficient
+# if you don't need to differentiate several regional versions of the same language.
+#
+# You need to make sure a locale corresponding to your language exists in the locales
+# supported by the React frontend.
+#
+# The order matters, because the first matching locale will be used if your LANGUAGES setting
+# declares languages without regions.
+REACT_LOCALES = getattr(settings, "REACT_LOCALES", ["en_US", "es_ES", "fr_FR", "fr_CA"])
+
+CATEGORIES_LOGO_IMAGE_WIDTH = getattr(settings, "CATEGORIES_LOGO_IMAGE_WIDTH", 216)
+CATEGORIES_LOGO_IMAGE_HEIGHT = getattr(settings, "CATEGORIES_LOGO_IMAGE_HEIGHT", 216)
```

Comment:
I don't understand these 2 lines, I don't think they are related to `i18n` work.

Reply:
I just brought them up from below for alphabetical sorting.

Reply:
Ok but all the file should be sort alphabetically, it's not the case.

Reply:
I don't think so it's just a few lines that are related to the same thing that should be in alphabetical order.